docs/integrations/aa/candide.md

```diff
@@ -0,0 +1,325 @@
+# Candide
+
+# Recover a Safe Account with Google using AbstractionKit
+
+This is the second of two guides on how to recovery a Safe Account using Google. In the first guide, we show the steps on how to add the recovery method. We recommend starting with the first guide [here](https://docs.candide.dev/wallet/guides/recovery-with-google-using-lit/).
+
+## What is AbstractionKit?
+AbstractionKit is a Typescript Library to easily build on Account Abstraction, with first class support for Safe Accounts. One of their use cases is to enable users to add recovery methods as a backup in case their use their main signer.
+
+To leverage the full potential of Account Abstraction, you can combine Lit with AbstractionKit to enable email / social recovery experience, while using a Smart Account as the smart wallet to sponsor gas for users, batch transactions, and more.
```

Comment:
> To leverage the full potential of Account Abstraction, you can combine Lit with AbstractionKit to enable email / social recovery experience, while using a Smart Account as the smart wallet to sponsor gas for users, batch transactions, and more.

Consider: "You can combine Lit with AbstractionKit to enable a powerful social recovery experience for your users while using Smart Accounts for gas sponsorship, transaction batching, and more."